src/native_background.ts

```diff
@@ -245,6 +246,17 @@ export async function temp(content: string, prefix: string) {
     return sendNativeMsg("temp", { content, prefix })
 }
 
+export async function winFirefoxRestart(profiledir: string) {
+    let current_version = await getNativeMessengerVersion()
+    let required_version = "0.1.6"
+
+    if (!await nativegate(required_version, false)) {
```

Comment:
Native gate checks the version for you, so you're checking it twice here. It matters because we think it's quite slow - #512 - although I've never actually tested it. If you just set the second argument to true it will tell the user anyway. It won't say the name of the command but it will be right after they tell it to `restart` so it should be quite obvious.

Reply:
I originally _"stole"_ this snippet from [`getenv`][0], which had a similar pattern of redundantly calling `getNativeMessengerVersion()`. I patched both of them [here][1].

Thanks.

[0]: https://github.com/gsbabil/tridactyl/commit/9ffc161b865d0ac6a5fe3712cc70cc74de3f2a98#diff-9ddd9172e4a54cdd30d56e5f10d5facaL274
[1]: https://github.com/gsbabil/tridactyl/commit/9ffc161b865d0ac6a5fe3712cc70cc74de3f2a98#diff-9ddd9172e4a54cdd30d56e5f10d5facaR253